modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/visualization_modes/table/Table.tsx

```diff
@@ -580,11 +583,21 @@ const EditFDSFieldModalContent = ({
 				<ClayForm.Group>
 					<ClayCheckbox
 						checked={fdsFieldSortable}
+						inline
 						label={Liferay.Language.get('sortable')}
 						onChange={({target: {checked}}) =>
 							setFSDFieldSortable(checked)
 						}
 					/>
+
+					<span
+						className="label-icon lfr-portal-tooltip ml-2"
+						title={Liferay.Language.get(
+							'if-enabled-this-checkbox-defines-whether-the-field-can-be-sorted'
```

Comment:
I'd probably rewrite this key a bit, following the `if-checked` pattern, and stating that what is sorted is not the field itself but rather the data set items. How would this sound?:
`if-checked,-data-set-items-can-be-sorted-by-this-field`

Reply:
Yeah that makes sense, thank you!
I'll wait for feedback from @pablo-agulla and @herrardo before making any changes.

Reply:
Made label changes.